prow/oss/config.yaml

```diff
@@ -264,6 +264,9 @@ gerrit:
   - org: https://linux-review.googlesource.com
     repos:
     - linux/kernel/git/torvalds/linux
+  - org: https://cloud-release-internal-review.googlesource.com
```

Comment:
I'm not sure that this prow instance has access to this Gerrit host. Since this PR is just for testing pubsub a GitHub repo should be sufficient, would suggest remove this entry and I'll take a look at which GitHub repo you can use for this one time test

Reply:
I will actually get rid of gerrit stuff, and just use periodic job.